tests/api/test_account.py

```diff
@@ -278,6 +255,52 @@ def test_who_can_see_user(
     assert content['data']['customers']['totalCount'] == 1
 
 
+ME_QUERY = """
+    query Me{
```

Comment:
The fact that it doesn't have a space before the bracket bugs me. It be more consistent if there was a space, if you can quickly do that change? Same for the query in `test_me_query_customer_can_not_see_note`'s body.
```suggestion
    query Me {
```